modules/irsa/main.tf

```diff
@@ -1,10 +1,7 @@
-data "aws_caller_identity" "aws_account" {}
```

Comment:
Why is this being removed?

Reply:
I added [this](https://github.com/dominodatalab/terraform-aws-eks/pull/206/files#diff-82f0360a5fa9e26b1a605c21d229da63678b0727c5e6a13bde75b6ebdc60b7d2R1) for flyte when I initially put flyte stuff in irsa. Now that I am moving flyte stuff to it's own submodule it is no longer necessary.